playground/.eslintrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  extends: '../.eslintrc.js',
+  env: {
+    node: true,
+    browser: true,
+  },
+  rules: {
+    'no-console': 'off',
+    'import/no-unresolved': 'off',
```

Comment:
Not sure why I'm getting errors from this rule 🤔 

But since it's only for the playground, I guess it might be okay to disable it here for now?

<img width="724" alt="Screenshot 2020-11-29 at 15 52 50" src="https://user-images.githubusercontent.com/3296643/100550950-f5bc6f00-327d-11eb-979d-3d7a3c23e018.png">

Reply:
Hm, that a bit weird. Did you install the node_modules in the playground folder?

Reply:
Haha of course, missed that 😅